tools/build-tools/src/buildVersion/buildVersion.ts

```diff
@@ -177,24 +244,21 @@ function main() {
     }
 
     // Generate and print the version to console
-    const version = getSimpleVersion(file_version, arg_build_num, arg_release, arg_patch);
+    const simpleVersion = getSimpleVersion(file_version, arg_build_num, arg_release, arg_patch);
+    const version = arg_test_build ? `0.0.0-${arg_build_num}-test` : simpleVersion;
     console.log(`version=${version}`);
     console.log(`##vso[task.setvariable variable=version;isOutput=true]${version}`);
-    if (arg_release) {
-        let isLatest = true;
-        if (arg_tag) {
-            const split = version.split(".");
-            if (split.length !== 3) {
-                console.error(`ERROR: Invalid format for release version ${version}`);
-                process.exit(8);
-            }
-            const tagName = `${arg_tag}_v${split[0]}.${parseInt(split[1]) + 1}.*`;
-            const out = child_process.execSync(`git tag -l ${tagName}`, { encoding: "utf8" });
-            if (out.trim()) {
-                isLatest = false;
-            }
-        }
-        console.log(`isLatest=${isLatest}`);
+
+    // Output the code version for test build
+    if (arg_test_build) {
+        const codeVersion = `${simpleVersion}-test`;
+        console.log(`codeVersion=${codeVersion}`);
+        console.log(`##vso[task.setvariable variable=codeVersion;isOutput=true]${codeVersion}`);
```

Comment:
where is this used?

Reply:
Looks like here:

https://github.com/microsoft/FluidFramework/blob/3bd905c997ba0431a07b8597ab709132c6f8db42/common/build/build-common/gen_version.js#L23-L26